scripts/friendly.coffee

```diff
@@ -0,0 +1,43 @@
+# coppied from https://www.npmjs.com/package/hubot-friendly since it was broken as a package
```

Comment:
friendly.coffee is added but removed from external-scripts?

Reply:
external scripts loads them from node-modules. the node modules marked hubot 2.x.x as required but they work with hubot 3.x.x so by copying them in I'm just avoiding dependency stuff.